tests/test_integrations/test_cybermindr.py

```
@@ -0,0 +1,137 @@
+        integration = sample_integration_dict("cybermindr", tokens)
+        service = integration_service_factory.get_service(None, integration)
+        actions = service.get_all_rest_api_actions()
+        for action in actions:
+            try:
+                task = sample_restapi_task(
+                    integration, action.code, action.parameters_definition
+                )
+                result = handle_task(task)
+                print(result.model_dump_json(indent=2))
+            except Exception:
+                traceback.print_exc()
```

Ruff (0.15.15)

[warning] `try`-`except` within a loop incurs performance overhead

(PERF203)